tests/data/upgrade_cost_analysis_generic.json

```diff
@@ -0,0 +1,76 @@
+{
+  "thermal_upgrade_params": {
+    "xfmr_upper_limit": 1.25,
+    "line_upper_limit": 1.25,
+    "line_design_pu": 0.75,
+    "xfmr_design_pu": 0.75,
+    "voltage_upper_limit": 1.05,
+    "voltage_lower_limit": 0.95
+  },
+  "voltage_upgrade_params": {
+    "initial_upper_limit": 1.05,
+    "initial_lower_limit": 0.95,
+    "final_upper_limit": 1.05,
+    "final_lower_limit": 0.95,
+    "target_v": 1,
+    "nominal_voltage": 120,
+    "nominal_pu_voltage": 1,
+    "tps_to_test": [],
+    "capacitor_sweep_voltage_gap": 1,
+    "reg_control_bands": [
+      1,
+      2
+    ],
+    "reg_v_delta": 0.5,
+    "max_regulators": 4,
+    "place_new_regulators": false,
+    "use_ltc_placement": false
+  },
+  "pydss_controllers": {
+    "pv_controller": {
+        "control1": "VVar",
+        "control2": "None",
+        "control3": "None",
+        "pf": 1,
+        "pf_min": 0.8,
+        "pf_max": 1.0,
+        "p_min": 0.0,
+        "p_max": 1.0,
+        "u_min": 0.94,
+        "u_db_min": 0.97,
+        "u_db_max": 1.03,
+        "u_max": 1.06,
+        "q_lim_pu": 0.44,
+        "pf_lim": 0.9,
+        "enable_pf_limit": false,
+        "u_min_c": 1.06,
+        "u_max_c": 1.1,
+        "p_min_vw": 10.0,
+        "vw_type": "Rated Power",
+        "percent_p_cutin": 10.0,
+        "percent_p_cutout": 10.0,
+        "efficiency": 100.0,
+        "priority": "Var",
+        "damp_coef": 0.8
+    }
+  },
+  "enable_pydss_controllers": true,
+  "include_pf1": true,
+  "upgrade_cost_database": "./disco/extensions/upgrade_simulation/upgrades/Generic_DISCO_cost_database_v2.xlsx",
+  "jobs": [
+    {
+      "model_type": "UpgradeCostAnalysisGenericModel",
+      "name": "p1uhs21_1247__p1udt5257__close__1__5",
+      "opendss_model_file": "upgrade-models/p1uhs21_1247/p1udt5257/PVDeployments/p1uhs21_1247__p1udt5257__close__1__5.dss",
+      "blocked_by": [],
+      "estimated_run_minutes": null
```

Comment:
Does DISCO also want an additional input for simulation start and end timestamp? 
We store this information in URBANopt and can pass this along.

@daniel-thom @kflemin

Reply:
We normally do require those as inputs. However, what we talked about last week is that DISCO will auto-select the time points based on the load shape profiles. We could change back, though.

Reply:
Since the jobs is an array of OpenDSS files I am imagining this is for separate URBANopt Scenarios. 
The pyDSS controllers are implemented at the project level - so the user preference is applied to all scenarios?

Reply:
Since the jobs is an array of OpenDSS files I am imagining this is for separate URBANopt Scenarios. 
The pyDSS controllers are implemented at the project level - so the user preference is applied to all scenarios?

Reply:
Yes, that is correct.